app/src/main/java/de/westnordost/streetcomplete/screens/main/map/components/StyleableOverlayMapComponent.kt

```diff
@@ -184,6 +184,7 @@ class StyleableOverlayMapComponent(
             .withProperties(
                 iconImage(get("icon")),
                 textField(get("label")),
+                textFont(arrayOf("Roboto Regular")),
```

Comment:
You had to write this because MapLibre uses some font by default?

Reply:
Yes, the default is `Open Sans Regular` and `Arial Unicode MS Regular`